app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -238,6 +240,11 @@ object TelemetryWrapper {
     }
 
     fun homeTileClickEvent(tile: HomeTile) {
+        if (tile.idToString() == YOUTUBE_TILE_ID) {
```

Comment:
We can double check, but this is taken directly from production FFTV code.  If there are issues with it, it will need to be changed in both.  I think it is important to have these two data stores be consistent and directly comparable with one another.